### ISSUE TITLE
Main story span

### ISSUE THREAD

**Issue by oliverjam** [bobbysebolao/learn-css-grid#1]
I think [`The grid item containing the main story image should span 2 columns`](https://github.com/bobbysebolao/learn-css-grid/blame/master/README.md#L23) should say `span 3 columns`? Since the "main story" should be a single containing element with both the image and text groups together.

**Comment by bobbysebolao** [comment]
I've placed the main story text and main story image as two separate grid items because rolling them both into one would require me to apply internal flexbox styling to that grid item. I think it would confuse things to involve flexbox in this challenge so I'd prefer to leave these main story grid items as is (unless you can think of a way to achieve the same layout that doesn't require extra CSS?)

**Comment by bobbysebolao** [comment]
![Screen Shot 2020-03-13 at 13 57 26](https://user-images.githubusercontent.com/30030755/76627262-9de4a900-6532-11ea-951d-47d88c194b04.png)

☝️ here's what it looks like when you put the text and image in the same grid item. You can fix it by applying `display: flex` to it, but I don't think this should be part of the challenge 

(update: I think subgrid would work too, but I don't think we want to introduce that since it's still not part of the main CSS spec?)

**Comment by oliverjam** [comment]
Yeah imo each separate "item" would be responsible for its own internal layout. The `main` grid would only lay the items themselves out. Can we not provide the internal item styles (e.g. the `display: flex`) so they don't have to worry about it?

**Comment by bobbysebolao** [comment]
I spent quite a while trying to implement the correct layout with a single main story span, but I can't get the grid items to align correctly.

![Screen Shot 2020-03-13 at 16 26 46](https://user-images.githubusercontent.com/30030755/76640231-7815cf00-6547-11ea-9b1f-d270e2297517.png)

Notice in the screenshot how the nested grid items don't align with the parent grid columns. I think it's because: 

1. The nested grid doesn't acknowledge the `grid-column-gap: 1rem` that I've set on the parent grid
2. The nested grid includes the parent grid column gap widths in calculating the width of its own columns
3. This results in columns that are slightly wider than those of the parent grid items, causing misalignment

Interestingly, turning the nested grid into subgrid with `grid-template-columns: subgrid;` fixes the issue in Firefox!

![Screen Shot 2020-03-13 at 16 35 49](https://user-images.githubusercontent.com/30030755/76641103-f58e0f00-6548-11ea-8a80-653d5a1a56e8.png)


Per the MDN docs on subgrid: 

> In the current specification, we would edit the above nested grid example to change the track definition of grid-template-columns: repeat(3, 1fr), to grid-template-columns: subgrid. The nested grid will then use the parent grid tracks to layout items.


**TL;DR:** I think it's probably safer to stick with the original implementation for now, given the lack of browser support for subgrid. What do you think @oliverjam ?

**Comment by oliverjam** [comment]
Yeah you've discovered the exact reason everyone can't wait for subgrid 😅 Maybe lets just stick with the original version yeah

**Comment by bobbysebolao** [comment]
Ok, I will close this issue then 👍
